RELEASING/changelog.py

```diff
@@ -108,21 +120,108 @@ def _get_github_login(self, git_log: GitLog) -> Optional[str]:
         self._github_login_cache[author_name] = github_login
         return github_login
 
+    def _has_commit_migrations(self, git_sha: str) -> bool:
+        commit = self._superset_repo.get_commit(sha=git_sha)
+        for file in commit.files:
+            if "superset/migrations/versions/" in file.filename:
+                return True
+        return False
+
+    def _get_pull_request_details(self, git_log: GitLog) -> Dict[str, Any]:
+        pr_number = git_log.pr_number
+        if pr_number:
+            detail = self._pr_logs_with_details.get(pr_number)
+            if detail:
+                return detail
+            else:
+                pr_info = self._fetch_github_pr(pr_number)
+
+        has_migrations = self._has_commit_migrations(git_log.sha)
+        title = pr_info.title if pr_info else git_log.message
+        pr_type = re.match(r"^(fix|feat|chore|refactor|docs|build|ci|/gmi)", title)
+        if pr_type:
+            pr_type = pr_type.group().strip('"')
+        else:
+            pr_type = None
```

Comment:
`pr_type` will already by `None` if it's false, so:

```suggestion
        if pr_type:
            pr_type = pr_type.group().strip('"')
```